src/Compiler/TypedTree/TypedTreePickle.fs

```diff
@@ -2510,6 +2536,40 @@ and u_modul_typ st =
           (u_qlist u_entity_spec) st
     ModuleOrNamespaceType(x1, x3, x5)
 
+and u_qualified_name_of_file st = 
+    let ident = u_ident st
+    QualifiedNameOfFile(ident)
+
+and u_pragma st =
+    let range = u_range st
+    let warningNumber = u_int st
+    ScopedPragma.WarningOff (range, warningNumber)
+
+and u_pragmas st =
+    u_list u_pragma st
+
+and u_checked_impl_file st = 
+    let qualifiedNameOfFile, pragmas, signature, hasExplicitEntryPoint, isScript =
+        u_tup5
+            u_qualified_name_of_file
+            u_pragmas
+            u_modul_typ
+            u_bool
+            u_bool
+            st
+
+    CheckedImplFile(
+        qualifiedNameOfFile,
+        pragmas,
+        signature,
+        // ModuleOrNamespaceContents needs implementing, feels hard but doable
```

Comment:
This is where the continuous progress should be happening.
Starting from most common types/expressions to the more rare ones.